docs/application/dotnet/guides/user-interface/nui/relative-layout.md

```diff
@@ -73,14 +82,533 @@ The `HorizontalAlignment` and the `VerticalAlignment` properties describe how to
+    BackgroundColor="White">
+
+    <View.Layout>
+        <RelativeLayout />
+    </View.Layout>
+
+</View>
+```
+
+</span>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+</div>
+
+### Red view
+
+The following code shows how to create redView with bounded by offsets.
+
+![RelativeLayoutExampleRedView](./media/relative_layout_example_red_view.png)
+
+<div id="TabSection2">
+    <div class="sampletab " id="ProjectCreateTab">
+        <button id="RelativeLayout-RedView-CSharp" class="tablinks " onclick="openTabSection(event, 'RelativeLayout-RedView-CSharp', 'TabSection2') ">C#</button>
+        <button id="RelativeLayout-RedView-Xaml" class="tablinks " onclick="openTabSection(event, 'RelativeLayout-RedView-Xaml', 'TabSection2') ">Xaml</button>
+    </div>
+    <div id="RelativeLayout-RedView-CSharp" class="tabcontent">
+        <table>
+            <tbody>
+                <tr>
+<span style="display:block">
+
+- `RelativeLayout.SetLeftRelativeOffset(redView, 0.0f)` - view's left is at the parent's left.
+- `RelativeLayout.SetRightRelativeOffset(redView, 0.1f)` - view's right is at the position of 0.1 width from the parent's left.
+- `RelativeLayout.SetTopRelativeOffset(redView, 0.0f)` - view's top is at the parent's top.
+- `RelativeLayout.SetBottomRelativeOffset(redView, 0.1f)` - view's bottom bound is at the position of 0.1 width from the parent's top.
+- `RelativeLayout.SetFillHorizontal=(redView, true)` and `SetRelativeLayout.FillVertical(redView, true)` - Since the view does not have its Width and Height,the view should be filled the relative position(left, right, top, bottom).
+
+```csharp
+View redView = new View
+{
+    BackgroundColor = Color.Red
+};
+
+RelativeLayout.SetLeftRelativeOffset(blueView, 0.0f);
+RelativeLayout.SetRightRelativeOffset(blueView, 0.1f);
+RelativeLayout.SetTopRelativeOffset(blueView, 0.0f);
+RelativeLayout.SetBottomRelativeOffset(blueView, 0.1f);
+
+RelativeLayout.SetFillHorizontal(redView, true);
+RelativeLayout.SetFillVertical(redView, true);
+
+relativeLayoutView.Add(redView);
+```
+
+</span>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+    <div id="RelativeLayout-RedView-Xaml" class="tabcontent">
+        <table>
+            <tbody>
+                <tr>
+<span style="display:block">
+
+- `RelativeLayout.LeftRelativeOffset="0.0"` - view's left is at the parent's left.
+- `RelativeLayout.RightRelativeOffset="0.1"` - view's right is at the position of 0.1 width from the parent's left.
+- `RelativeLayout.TopRelativeOffset="0.0"` - view's top is at the parent's top.
+- `RelativeLayout.BottomRelativeOffset="0.1"` - view's bottom bound is at the position of 0.1 width from the parent's top.
+- `RelativeLayout.FillHorizontal="true"` and `RelativeLayout.FillVertical="true"` - Since the view does not have its Width and Height,the view should be filled the relative position(left, right, top, bottom).
```

Comment:
```suggestion
- `RelativeLayout.FillHorizontal="true"` and `RelativeLayout.FillVertical="true"` - Since the view does not have its Width and Height, the view should be filled the relative position(left, right, top, bottom).
```

Reply:
This comment is not applied. Would you please check?